06_create_cluster.sh

```diff
@@ -83,4 +83,8 @@ if [[ ! -z "${ENABLE_METALLB}" ]]; then
 	popd
 fi
 
+if [[ ! -z "${ENABLE_VIRTUAL_MEDIA_VIA_EXTERNAL_NETWORK}" ]]; then
```

Comment:
Nit (and I know this is a common pattern in dev-scripts, but still): ! -z is the same as -n.